js/ResourcesView.js

```diff
@@ -0,0 +1,51 @@
+import Adapt from 'core/js/adapt';
+export default class ResourcesView extends Backbone.View {
```

Comment:
```suggestion
import Adapt from 'core/js/adapt';

export default class ResourcesView extends Backbone.View {
```

Add line to differentiate between import and export statements.